src/src/PluginStructs/P128_data_struct.cpp

```diff
@@ -653,84 +644,94 @@ bool P128_data_struct::plugin_write(struct EventStruct *event,
     NeoPixelSendStatus(event);
   }                                                                            // command neopixel
 
-  if (speed == 0) { mode = On;                                                 // speed = 0 = stop mode
+  if (speed == 0) { mode = P128_modetype::On;                                  // speed = 0 = stop mode
   }
   speed    = (speed > SPEED_MAX || speed < -SPEED_MAX) ? defaultspeed : speed; // avoid invalid values
   fadetime = (fadetime <= 0) ? 20 : fadetime;
 
   return success;
 }
 
+void P128_data_struct::rgb2colorStr() {
+  colorStr.clear();
+  rgb.R < 16 ? colorStr  = F("0") : F("");
```

Comment:
Why not like this?
```c++
if (rgb.R < 16) colorStr += '0';
```
Then you don't need to load flash strings, which is rather expensive if you need to do it often.